doc/modules/multiclass.rst

```diff
@@ -425,3 +427,13 @@ averaged together.
 
     Jesse Read, Bernhard Pfahringer, Geoff Holmes, Eibe Frank,
         "Classifier Chains for Multi-label Classification", 2009.
+
+.. regressorchain:
```

Comment:
needs ``_`` in the front

Reply:
Should I submit a new PR with these changes?

Reply:
yeah, sounds good. And mention that in #10258 and check the circle CI build whether the links work. Maybe also add a "see also" from the regressor to the classifier and vice versa.